aws-transfer-user/src/test/java/software/amazon/transfer/user/ListHandlerTest.java

```diff
@@ -0,0 +1,70 @@
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import software.amazon.awssdk.services.transfer.model.HomeDirectoryType;
+import software.amazon.awssdk.services.transfer.model.ListUsersRequest;
+import software.amazon.awssdk.services.transfer.model.ListUsersResponse;
+import software.amazon.awssdk.services.transfer.model.ListedUser;
+import software.amazon.awssdk.services.transfer.model.ThrottlingException;
+import software.amazon.cloudformation.proxy.OperationStatus;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+@ExtendWith(MockitoExtension.class)
+public class ListHandlerTest extends AbstractTestBase {
+
+    private final ListHandler handler = new ListHandler();
+
+    @Test
+    public void handleRequest_SimpleSuccess() {
+
+        final ResourceModel model = ResourceModel.builder().build();
+
+        final ResourceHandlerRequest<ResourceModel> request =
+                ResourceHandlerRequest.<ResourceModel>builder().desiredResourceState(model).build();
+
+        ListUsersResponse listUsersResponse =
+                ListUsersResponse.builder()
+                        .users(
+                                ListedUser.builder()
+                                        .arn("testArn")
+                                        .userName("testUser")
+                                        .homeDirectoryType(HomeDirectoryType.PATH)
+                                        .homeDirectory("/")
+                                        .sshPublicKeyCount(1)
+                                        .role("role")
+                                        .build())
+                        .build();
+        doThrow(ThrottlingException.builder().build())
+                .doReturn(listUsersResponse)
+                .when(sdkClient)
+                .listUsers(any(ListUsersRequest.class));
+
+        ProgressEvent<ResourceModel, CallbackContext> response =
+                handler.handleRequest(proxy, request, new CallbackContext(), proxyClient, logger);
+
+        assertThat(response).isNotNull();
+        assertThat(response.getStatus()).isEqualTo(OperationStatus.IN_PROGRESS);
+        assertThat(response.getCallbackDelaySeconds()).isEqualTo(THROTTLE_CALLBACK_DELAY_SECONDS);
+
+        // Call again in response to ThrottleException
+        response =
+                handler.handleRequest(proxy, request, new CallbackContext(), proxyClient, logger);
+
+        assertThat(response).isNotNull();
+        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
+        assertThat(response.getCallbackContext()).isNull();
+        assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
+        assertThat(response.getResourceModel()).isNull();
+        assertThat(response.getResourceModels()).isNotNull();
```

Comment:
Can we make a stronger assertion here that the models are what we expected rather than just not null?